test/cert-manager-v1alpha1/test.sh

```diff
@@ -30,70 +30,57 @@ validateMetrics() {
     fi
 }
 
-export KUBECONFIG=""
-K3D_NAME=cert-exporter
-CONFIG_PATH="k3d get-kubeconfig --name=$K3D_NAME"
+KIND_CLUSTER_NAME=cert-exporter
 
-k3d create --name=$K3D_NAME
-echo -n "Ensuring k3d is running..."
-while true; do
-  k3d list 2>&1 | grep ".*$K3D_NAME.*running" >/dev/null && echo "done" && break \
-    || (echo -n . && sleep 1)
-done
+echo -n "Create cluster"
+kind create cluster --name=$KIND_CLUSTER_NAME
 
-echo -n "Getting kubeconfig..."
-while true; do
-  eval $CONFIG_PATH 2>&1 | grep "$K3D_NAME/kubeconfig.yaml" >/dev/null && echo done && break \
-    || (echo -n . && sleep 1)
-done
-echo Config is available at $(eval $CONFIG_PATH)
+echo -n "Get kubeconfig"
+kind export kubeconfig --name=$KIND_CLUSTER_NAME
 
-kubectl --kubeconfig $(eval $CONFIG_PATH) create namespace cert-manager
-kubectl --kubeconfig $(eval $CONFIG_PATH) label namespace cert-manager certmanager.k8s.io/disable-validation=true
-kubectl --kubeconfig $(eval $CONFIG_PATH) apply -f https://github.com/jetstack/cert-manager/releases/download/v0.10.1/cert-manager.yaml
+kubectl --kubeconfig=$HOME/.kube/config create namespace cert-manager
```

Comment:
Can we get kind to use a different kubeconfig?  I would prefer it doesn't make changes to my default kubeconfig as all my standard cluster configs are in there.

Reply:
Specify the kubeconfig path when creating a cluster. PTAL :bow: